dev/ci/dev-env.Dockerfile

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright 2023 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+FROM openjdk:8-jdk-buster
+LABEL maintainer="support@datastrato.com"
+
+WORKDIR /root/gravitino
+
+COPY ../docker/dockerfile /root/gravitino
```

Comment:
What is this command trying to copy?

Reply:
that's should be COPY . /root/gravitino
I write wrong